src/libOpenImageIO/imagebuf.cpp

```diff
@@ -1418,8 +1418,20 @@ ImageBuf::write(ImageOutput* out, ProgressCallback progress_callback,
             int chunk = clamp(round_to_multiple(int(budget / slsize), 64), 1,
                               1024);
             std::unique_ptr<char[]> tmp(new char[chunk * slsize]);
+
+            const bool isDecreasingY = !strcmp(out->format_name(), "openexr")
+                                       && outspec.get_string_attribute(
+                                              "openexr:lineOrder")
+                                              == "decreasingY";
+            const int yStart = isDecreasingY
+                                   ? ((outspec.height - 1) / chunk) * chunk
+                                   : 0;
+            const int yDelta = isDecreasingY ? -chunk : chunk;
             for (int z = 0; z < outspec.depth; ++z) {
-                for (int y = 0; y < outspec.height && ok; y += chunk) {
+                for (int y = yStart; ((isDecreasingY && y >= 0)
+                                      || (!isDecreasingY && y < outspec.height))
+                                     && ok;
+                     y += yDelta) {
```

Comment:
This is a little awkward. Why not 

```
            const int yStart = isDecreasingY
                                   ? ((outspec.height - 1) / chunk) * chunk
                                   : 0;
            const int yDelta = isDecreasingY ? -chunk : chunk;
            const int yEnd = yStart + yDelta;
            for (int z = 0; z < outspec.depth; ++z) {
                for (int y = yStart; y != yEnd && ok; y += yDelta) {
```

Reply:
Agreed. I updated the code to something similar yEnd = yStart + numChunks * yDelta which is I believe what you were getting at.

I also change yStart and yEnd to yLoopStart and yLoopEnd because there were already yend variables near this code and having folks try to keep the difference between yend and yEnd straight just seems like asking for bugs.

Reply:
Yeah, sorry, I think I was on an airplane when I wrote that, don't take what I say tooooo literally.